### DISCUSSION TITLE
Run the multiplexing daemon without installing Wezterm

### DISCUSSION THREAD

**Discussion by abdelbk** [wezterm/wezterm#6382] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Hi,

I'm trying to use Wezterm multiplexing on a remote Linux server to which I don't have full permissions.
The distribution has neither X11 nor Wayland, so I'm not able to build Wezterm from source.

Is there a way to run the Wezterm multiplexing daemon without building the whole wezterm project ?
I think it's handled by wezterm-mux-server, but I don't seem to find a way to build it separately.

Is it achievable ?

Thank you,

**Comment by loops** [comment, marked as answer]
If you're trying to build from source you can build "wezterm-mux-server", and the "wezterm" executable which you'll also need :

    cargo build --jobs 2 --release --bin wezterm-mux-server --bin wezterm

This will avoid building the wezterm-gui executable, which isn't required on the remote server.   You can find the binaries in the "target/release" directory.

**Comment by wez** [comment]
Compiling your own is likely to have the best compatibility with your system, but: Ubuntu binaries can be found in a tarball (not in a deb) on the "Raw" section of the download page here: https://wezfurlong.org/wezterm/install/linux.html#__tabbed_1_10

**Comment by abdelbk** [comment]
Thank you @loops @wez ! It works great
